src/NzbDrone.Core/NzbDrone.Core.csproj

```diff
@@ -1127,6 +1131,7 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="App.config" />
+    <None Include="ClassDiagram1.cd" />
```

Comment:
This line should not be included, it was a class diagram I temporarily created and I thought I deleted it all.